### PR TITLE
first draft of microservice dockerfile

### PR DESCRIPTION
**Pull request by zancas**
@pjbreaux 
This is the first change necessary to roll out the dockerization of our ansible server.

The Dockerfile is a the spec for the our ansible microservice.

The new install of git (with yum) is the minimum change we needed to make to the controller to install the agent and driver.

### CODE REVIEW

**Comment by zancas** [comment]
I tested these changes by pointing my development branches at my fork of this repo, and running the tlc commands.

**Comment by zancas** [comment]
@pjbreaux it looks like I don't have the authority to assign you...

File: playbooks/roles/install_lbaasv2_driver/tasks/main.yml [hunk 1]
```diff
@@ -21,6 +21,11 @@
     state: present
   when: not use_pip|bool
 
+- name: Install git
+  package:
+    name:  git-1.8.3.1-6.el7_2.1
```
Comment (pjbreaux) [inline playbooks/roles/install_lbaasv2_driver/tasks/main.yml]: I think this should be reference via a default variable, which would be added here:

https://github.com/f5devcentral/f5-openstack-ansible/blob/liberty/playbooks/roles/configure_lbaasv2_driver/vars/main.yml

And the user can override in their playbook if they desire.

File: playbooks/roles/install_lbaasv2_driver/tasks/main.yml [hunk 2]
```diff
@@ -21,6 +21,11 @@
     state: present
   when: not use_pip|bool
 
+- name: Install git
+  package:
+    name:  '{{ versioned_git_yum_package }}'
```
Comment (pjbreaux) [inline playbooks/roles/install_lbaasv2_driver/tasks/main.yml:26]: How did the test for this change go?

**Pull request merged**